### DISCUSSION TITLE
Version 2.987

### DISCUSSION THREAD

**Discussion by gzsombor** [SkyTubeTeam/SkyTube#1206] [General]
* Fix 'unable to download videos' error, due to recent Youtube changes
* Fix error because of missing channel banner/avatar
* Fix crash happening, when SponsorBlock was not enabled when video was downloaded, however enabled later
* Fix missing context menu items.
* Refactor to use the latest NewPipe playlist fecthing code
* update style of screenshots - Tomoya Kuroda
* Updated translations for:
    * Czech - Woyta, Not Fill
    * Chinese (Simplified) - Eric
    * French - Ninjdai
    * German - RandomModderJDK
    * Hindi - ShareASmile
    * Hungarian - Szia Tomi
    * Italian - Massimo Pissarello
    * Japanese - Takaki IEKURA
    * Polish - Michal L
    * Portugeese (Brazil) - gkubya, Yuri Argolo, Flavio F. M, Carlos Luiz
    * Portugeese - SACAMOTO SEIJI
    * Punjabi - ShareASmile
    * Russian - Alexey Esaulenko, jooblex

<hr /><em>This discussion was created from the release <a href='https://github.com/SkyTubeTeam/SkyTube/releases/tag/v2.987'>Version 2.987</a>.</em>

**Comment by bha4** [comment]
Please fix the video players most of the time The video is not playing.

**Comment by cheflepardon** [comment]
The following are the bugs I have encountered while using Skytube Extra version 2.987.
1.The app crashes the moment I cast any video to my smart TV , the TV plays the video I casted but the app crashes and doesn't not give me any buttons to navigate the video I casted on screen.
2. After opening the application the loading symbol doesn't go for long time and just rotates for no reason as I have turned off all the tabs from the main screen not sure what the application is trying to load in the background.
3. The app gets crashed after casting the video to screen and if we open the application the casting icon showcases that app is disconnected from TV where we casted the video. If we click again on the casting icon it gets connected and again crashed , however all this while the video which we cast will be playing on TV.
4. The app after crashing the notification bar of the phone shows that a video is getting casted by showing the name of the video and it shows the seeking buttons and volume buttons on the notification bar, however if we try doing something on that by clicking on it then it crashes again.
5. Occasionally the notification bar also showcases an icon of phone apart from volume button and if clicked and connected to TV by putting in code displayed on TV the phone turns out to become a navigation remote to forward video, volume button etc but this option is not on displayed everything but occasionally not sure if this function is enabled in skytube extra or Google inbuilt feature for android users.
6. The if not casted to TV if trying to use independently crashes after few minutes as initially it tries to load the video but crashed ..
7. This version of skytube extra is full with bugs and please try to bring back the normal functionality where we connect to TV by clicking on the cast icon and TV used to display as skytube extra and post that we used to navigate on the app by playing a video and application was showing the buttons to forward or reverse the video and play and pause buttons and full control of the video and TV used to be on phone itself. Please fix this issue and bring back that feature .. now it's just crashing for anything and everything.. expecting the fix of this issues on urgency.. thanks in advance..

**Reply by cheflepardon** [reply]
@gzsombor please address this issue as app is crashing heavily now not able to even navigate through app when casting now
